coralproject/talk
# Yarn fixes for Docker

## What does this PR do?

Temporarily patches upstream issue with the node:7.6 docker image: https://github.com/nodejs/docker-node/issues/346

## How do I test this PR?

- Run: `docker build -t coralproject/talk .` from the root of the project after running `yarn build`
- Ensure it completed with status code 0
